src/azure-cli/azure/cli/command_modules/acr/import.py

```diff
@@ -94,18 +95,23 @@ def acr_import(cmd,
                                               target_tags=target_tags,
                                               untagged_target_repositories=repository,
                                               mode=ImportMode.force.value if force else ImportMode.no_force.value)
+
     result_poller = client.import_image(
         resource_group_name=resource_group_name,
         registry_name=registry_name,
         parameters=import_parameters)
 
+    if no_wait:
+        return result_poller
+
     return _handle_result(cmd, result_poller, source_registry, source_image, registry)
 
 
 def _handle_result(cmd, result_poller, source_registry, source_image, registry):
     from msrestazure.azure_exceptions import ClientException
     try:
-        result = LongRunningOperation(cmd.cli_ctx, 'Importing image...')(result_poller)
+        result = LongRunningOperation(cmd.cli_ctx, 'Importing image...', 'Import has finished')(result_poller)
```

Comment:
maybe consider:
```suggestion
        result = LongRunningOperation(cmd.cli_ctx, 'Importing image...', 'Import succeeded')(result_poller)
```
or something else like "Import succeeded" or "Import completed" or just "success".